src/main/java/in/hp/java/bookservice/BookServiceApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
 
 @SpringBootApplication
+@EnableCircuitBreaker
```

Comment:
It is not discovery annotation.

Reply:
Discovery Annotation is not needed with the latest version of spring boot and cloud.